.gitignore

```diff
@@ -44,6 +44,4 @@ vignettes/*.pdf
 Desktop.ini
 *.icloud
 
-.Rproj.user
```

Comment:
I think .Rproj.user is included in the default gitignore as created by github for R projects and probably, so I would rather not change this. Or rather, could you check if github includes it per default, if not I will approve

Reply:
I agree: .gitignore needs an entry with ".Rproj.user" and there is one already in line 20. This one is a duplicate.

Reply:
alright